src/components/assets/AssetCard.vue

```diff
@@ -67,12 +67,15 @@ export default {
       type: Object
     }
   },
-  data: () => ({
-    imgNotFound: require('@/assets/img-not-found.png')
-  }),
+  data: function() {
+    return {
+      imgNotFound: require('@/assets/img-not-found.png'),
+      thumbnailUrl: this.asset.thumbnailUrl
+    }
+  },
```

Comment:
@pauline2k - Amended. You think this will work?

Reply:
Let’s try it!